packages/mui-lab/src/themeAugmentation/index.d.ts

```diff
@@ -1,3 +1,5 @@
 export * from './overrides';
 export * from './props';
 export * from './components';
+
+export * from '@mui/x-date-pickers/themeAugmentation';
```

Comment:
```suggestion
```
I think it should not appear here, otherwise `@mui/x-date-pickers` is a dependency of `@mui/lab`?

Reply:
I don't remember who was in on the meeting, but we decided to re-export the MIT pickers from `@mui/x-date-pickers` in the lab, to make the migration smoother for users (they can update the lab for the other component without having to migrate the pickers at the same time).

So `@mui/lab` has `@mui/x-date-pickers` as a dependecy.

Reply:
I see, thanks for explaining.

Reply:
We should probably explain that in the blog post I started